q2_alignment/_filter.py

```diff
@@ -79,4 +79,19 @@ def mask(alignment: skbio.TabularMSA, max_gap_frequency: float=0.05,
                                                    min_conservation)
     combined_mask = gap_mask & conservation_mask
     # apply the mask and return the resulting alignment
-    return _apply_mask(alignment, combined_mask)
+    result = _apply_mask(alignment, combined_mask)
+
+    if result.shape[1] == 0:
+        num_input_positions = alignment.shape[1]
+        percent_passed_gap_filter = \
+            (gap_mask.sum() / num_input_positions) * 100.
+        percent_passed_conservation_filter = \
+            (conservation_mask.sum() / num_input_positions) * 100.
+        raise ValueError("No alignment positions remain after filtering. The "
+                         "filter thresholds will need to be relaxed. %1.2f%% "
```

Comment:
This may be cool to use: `'{percent:.2%}'.format(percent=1/3)`

Then you don't have to multiply by 100 in two places and it's a little clearer what's happening.